spec/jekyll-last-modified-at/date_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Date Tag' do
+  context 'A committed post file' do
+    def setup(file, layout)
+      @post = setup_post(file)
+      do_render(@post, layout)
+    end
+
+    it 'has date' do
+      setup('no-date.md', 'date_with_format.html')
```

Comment:
Currently struggling with:

```
  1) Date Tag A committed post file has date
     Failure/Error: post.output = Renderer.new(@site, post, @site.site_payload).run
     
     Liquid::SyntaxError:
       Liquid syntax error (line 6): Unknown tag 'page'
```

Removing `page.` from `spec/fixtures/_layouts/date_with_format.html` (so it's just `date`) doesn't change anything. I'm probably missing something obvious but it's tired and I'm late.